docs/tools/terraform/reference/cookbook/m3db-m3agg-recipe.rst

```diff
@@ -0,0 +1,115 @@
+Monitor PostgreSQL® metrics using M3-related services
+=====================================================
+
+Aiven for M3DB is a powerful metrics engine that can be used to monitor your Aiven services on a very large scale. Aiven for M3 Aggregator can store your data at various resolutions for different workloads at scale. You can monitor a relational database like PostgreSQL by sending its metrics to M3DB with M3 Aggregator integration and then visualizing the metrics data on an Aiven for Grafana® dashboard.
```

Comment:
Is it an engine or a database? Also, you're using it only for the monitoring of Aiven services but can be used for any time series data scenario

```suggestion
Aiven for M3DB is a powerful metrics database that can be used to monitor your Aiven services on a very large scale. Aiven for M3 Aggregator can store your data at various resolutions for different workloads at scale. You can monitor a relational database like PostgreSQL by sending its metrics to M3DB with M3 Aggregator integration and then visualizing the metrics data on an Aiven for Grafana® dashboard.
```

Reply:
Hi @ftisiot good point. Using PG as source made it very similar to the other recipe. I'll change the source.

On the "engine" or "database", I took the wording from the landing page of https://m3db.io. I think metrics engine or time-series database - both are Ok. But I can swap metrics engine with time-series database if you prefer.

Reply:
Similar wording is found here: https://www.youtube.com/watch?v=YKXVxbvWW3k

I believe that M3, as a whole, is a metrics engine whereas M3DB (a subset) is the time-series database.